### PR TITLE
Move ASP.NET Core sample to csproj

### PR DESCRIPTION
**Pull request by MarcinHoppe**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@gbiellem (no rush but...) what bits did u tick when u installed 2017? 

Seems we r missing the web sdk
```
C:\BuildAgent\work\273e95acce0e59c5\samples\web\send-from-aspnetcore-webapi\Core_6\WebApplication\WebApplication.csproj : error MSB4019: The imported project "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\Sdks\Microsoft.NET.Sdk.Web\Sdk\Sdk.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
```


confirmed on the build agent
image

might be related to this http://stackoverflow.com/questions/41542151/unable-to-import-sdk-into-dotliquid-build-in-visual-studio-2017rc

**Comment by SimonCropp** [comment]
@MarcinHoppe until the above is sorted i think just exclude this solution explicit from the TC build script for the samples. ie in this case "not building on TC" should not block you fixing this samples

**Comment by gbiellem** [comment]
@simoncropp not sure. Logon to the agents as admin and rerun the vs 2017
install tool from the start menu and it will tll you what's installed. Add
whatever you need to the agents and i'll reimage next week when i'm back.
I'm at Universal Studios @ Sentosa today, so no laptop, so cant help more
than that

**Comment by SimonCropp** [comment]
@gbiellem thanks. done

@MarcinHoppe your build is now green

**Comment by MarcinHoppe** [comment]
Cool @SimonCropp. Thanks a lot!

File: samples/web/send-from-aspnetcore-webapi/sample.md [hunk 1]
```diff
@@ -2,88 +2,20 @@
...
+reviewed: 2017-04-27
 related:
 - nservicebus/hosting
 ---
 
 
 This sample shows how to send messages to an NServiceBus endpoint from an ASP.NET Core WebAPI application. 
 
-WARNING: This sample runs on the full .NET framework 4.6.1 but utilizes both .NET Core and .NET Framework dependencies, i.e. ASP.NET Core and NServiceBus. So while .NET Core dependencies can be consumed by the .NET Framework runtime, the .NET Core runtime is not currently supported.
+WARNING: This sample runs on the full .NET Framework 4.6.1 but utilizes both .NET Standard and .NET Framework dependencies, i.e. ASP.NET Core and NServiceBus. So while .NET Standard dependencies can be consumed by the .NET Framework runtime, the .NET Core runtime is not currently supported.
```
Comment (SimonCropp) [inline samples/web/send-from-aspnetcore-webapi/sample.md]: should we now mention VS 2017 is required?

Comment (SimonCropp) [inline samples/web/send-from-aspnetcore-webapi/sample.md]: perhaps in pre-reqs?

Comment (MarcinHoppe) [inline samples/web/send-from-aspnetcore-webapi/sample.md]: Fair point. I will add information about VS 2017 to pre-reqs and also to the info about the project template that was used.

File: samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/WebApplication.csproj [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="1.1.1" />
+    <PackageReference Include="NServiceBus" Version="6.2.1" />
+  </ItemGroup>
```
Comment (SimonCropp) [inline samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/WebApplication.csproj]: do we no longer need a ref to Kestrel?

Comment (MarcinHoppe) [inline samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/WebApplication.csproj]: It seems to be added transitively by:

```xml
<PackageReference Include="Microsoft.AspNetCore" Version="1.1.1" />
```

We still have a call to `UseKestrel()` in the as well as `UseIISIntegration()` so nothing has changed here.

Comment (SimonCropp) [inline samples/web/send-from-aspnetcore-webapi/Core_6/WebApplication/WebApplication.csproj]: weird. ok

**Pull request merged**